### PR TITLE
support jetson-nano 4gb B01 model

### PR DESCRIPTION
**Pull request by ksachdeva**
Hi,

Thanks for this great work.

There are two nano dev kits - A02 and B01. Mostly if you purchase now then you will only get B01. I have B01 from more than 1 year now.

At present, the argument to jetson-image-creator in your script is taking 200. This is valid for A02 but for B01 it should 300.

I have taken the liberty to add support by changing the requirement for the value of the JETSON_NANO_BOARD environment variable.

I understand that unfortunately, this change will break your blog post.

Regards
Kapil

### CODE REVIEW

File: create-image.sh
Comment (pythops) [inline create-image.sh:77]: Hi @ksachdeva 

Thanks for your contributions.

I think it's great idea to support the new Jetson boards. However, I would suggest some changes so we support all the different revisions of the boards. 

I would go with defining a new optional env variable `JETSON_NANO_REVISION` and this variable could take `200` or `300` values or any valid values for the revision (should take a look at the nvidia doc to see what are the valid values). I would suggest to have `200` for the default as I assume this revision is more common. And then the script will be:
```
./jetson-disk-image-creator.sh -o jetson.img -b jetson-nano -r  $JETSON_NANO_REVISION
```
Once those changes are done, then I can merge it and update my blog post

**Review by pythops — CHANGES_REQUESTED** [review]

File: create-image.sh
Comment (ksachdeva) [reply create-image.sh]: I have pushed the requested changes.  I have kept the default to 200 as per your recommendation however my understanding is that the board rev 300 is now more common and it is the only one Nvidia makes but I have not done extensive research on it.

File: create-image.sh
Comment (pythops) [inline create-image.sh:69]: Probably better if you change it to
```
printf "Create image for Jetson nano 4GB board ($nano_board_rev revision)
```

File: create-image.sh
Comment (pythops) [inline create-image.sh:67]: You can remove this comment. It's unnecessary in my opinion.

File: create-image.sh
Comment (pythops) [inline create-image.sh:69]: Please add the word `revision` as well.
```
 printf "Create image for Jetson nano 4GB board ($nano_board_rev revision)"
 ```

File: create-image.sh
Comment (pythops) [inline create-image.sh:73]: Remove the trailing space here

File: create-image.sh
Comment (pythops) [inline create-image.sh:76]: These lines should be aligned with the ones above.

**Comment by ksachdeva** [comment]
Hi @pythops,

I think it will be more productive for you to make these changes yourself

Thanks for this great work

Regards
Kapil

**Comment by pythops** [comment]
@ksachdeva 
I pushed the changes on GitHub and updated the blog post

**Pull request closed without merging**
